### PR TITLE
Feature(KonaFiber):add iotdb stress test

### PR DESCRIPTION
**Pull request by jun0315**
add iotdb stress test
（SyncDatabaseDemo.java and ConnectionPool.java code from Miao Zheng write https://github.com/Tencent/TencentKona-8/tree/KonaFiber/demo/fiber/mysql_sync_stress_demo）

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/TencentKona-8?pullRequest=50) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/TencentKona-8?pullRequest=50) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/TencentKona-8?pullRequest=50) it.</sub>

File: demo/fiber/iotdb-sync-stress-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+dependency:
+1. iotdb
```
Comment (huishi-hs) [inline demo/fiber/iotdb-sync-stress-demo/README.md:2]: Can you update iotdb's server setup process here?

Comment (jun0315) [reply demo/fiber/iotdb-sync-stress-demo/README.md:2]: Okay, tonight I will supplement the creation process of the iotdb instance in as much detail as possible

Comment (jun0315) [reply demo/fiber/iotdb-sync-stress-demo/README.md]: I have updated iotdb's server and cli setup process in the new commit

**Comment by miao-zheng** [comment]
Is iotdb a synchronized database like mysql? 
Does iotdb have asynchronized interface？ If have，you should write the demo like netty_stress_test.

File: demo/fiber/iotdb-sync-stress-demo/pom.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>iotdb-session</artifactId>
+            <version>0.12.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>iotdb-jdbc</artifactId>
+            <version>0.12.0</version>
+        </dependency>
+    </dependencies>
```
Comment (huishi-hs) [inline demo/fiber/iotdb-sync-stress-demo/pom.xml:25]: there is no jar-with-dependencies configure in pom,  target/iotdb-sync-stress-demo-1.0-SNAPSHOT-jar-with-dependencies.jar will not be generated.

File: demo/fiber/iotdb-sync-stress-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+> sbin\start-cli.bat -h 127.0.0.1 -p 6667 -u root -pw root
+```
+
+test steps:
+1. create DATABASE zm;(execute "SET STORAGE GROUP TO root.zm")
+2. insert data into root.zm;(execute "INSERT INTO root.zm.wf01.wt01(timestamp,status,temperature) values(200,false,20.71)");
+3. execute "java -jar target/iotdb-sync-stress-demo-1.0-SNAPSHOT-jar-with-dependencies.jar 1000 100000 0"(1000 means thread num, 100000 means request count, 0 means use fiber)
+
+
+test result:
```
Comment (huishi-hs) [inline demo/fiber/iotdb-sync-stress-demo/README.md:30]: could you please share platform collect performance result.

**Comment by jun0315** [comment]
> Is iotdb a synchronized database like mysql?
> Does iotdb have asynchronized interface？ If have，you should write the demo like netty_stress_test.

@miao-zheng  Thanks for you review!
Iotdb is also a synchronized database like mysql, it doesn't have asynchronized interface. 
I will try to use a java client that supports asynchrony to access the database with high concurrency such as mybatis.

File: demo/fiber/iotdb-sync-stress-demo/pom.xml (hunk 2 above)
Comment (jun0315) [reply demo/fiber/iotdb-sync-stress-demo/pom.xml:25]: @huishi-hs Thanks for you review!

I have updated the new pom file and will generate iotdb-sync-stress-demo-1.0-SNAPSHOT-jar-with-dependencies.jar.

File: demo/fiber/iotdb-sync-stress-demo/README.md (hunk 3 above)
Comment (jun0315) [reply demo/fiber/iotdb-sync-stress-demo/README.md:30]: I have updated my test device information in the new commit.

**Comment by huishi-hs** [comment]
LGTM

**Pull request merged**